### PR TITLE
fix: event record scheme error

### PR DESCRIPTION
**Pull request by gxthrj**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description
#468 

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: pkg/ingress/controller.go [hunk 1]
```diff
@@ -37,6 +38,7 @@ import (
 	"github.com/apache/apisix-ingress-controller/pkg/apisix"
 	"github.com/apache/apisix-ingress-controller/pkg/config"
 	"github.com/apache/apisix-ingress-controller/pkg/kube"
+	apisixScheme "github.com/apache/apisix-ingress-controller/pkg/kube/apisix/client/clientset/versioned/scheme"
```
Comment (tokers) [inline pkg/ingress/controller.go:41]:
```suggestion
	apisixscheme "github.com/apache/apisix-ingress-controller/pkg/kube/apisix/client/clientset/versioned/scheme"
```

Basically, all characters are in the small case as the import package name alias.

File: pkg/ingress/controller.go [hunk 2]
```diff
@@ -166,6 +168,7 @@ func NewController(cfg *config.Config) (*Controller, error) {
 	}
 
 	// recorder
+	utilruntime.Must(apisixScheme.AddToScheme(scheme.Scheme))
```
Comment (tokers) [inline pkg/ingress/controller.go:171]: I think to check the return `err` from `apisixscheme.AddToScheme` and return once `err` is not `nil` is better, basically, we don't panic the program if internal errors happen in the same context.

Comment (gxthrj) [reply pkg/ingress/controller.go:171]: The official example I refer to : https://github.com/kubernetes/sample-controller/blob/master/controller.go#L97

File: pkg/ingress/controller.go (hunk 1 above)
Comment (gxthrj) [reply pkg/ingress/controller.go]: Done

File: pkg/ingress/controller.go (hunk 2 above)
Comment (tokers) [reply pkg/ingress/controller.go]: OK, not a big deal.

**Review by tokers — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**